storage/bootstrap/bootstrapper/commitlog/options.go

```diff
@@ -61,3 +65,23 @@ func (o *options) SetCommitLogOptions(value commitlog.Options) Options {
 func (o *options) CommitLogOptions() commitlog.Options {
 	return o.commitLogOpts
 }
+
+func (o *options) SetM3TSZEncodingConcurrency(value int) Options {
+	opts := *o
+	opts.m3TSZEncodingConcurrency = value
+	return &opts
+}
+
+func (o *options) M3TSZEncodingConcurrency() int {
```

Comment:
Lets just call this `EncodingConcurrency` since all the encoder/decoder interfaces are generic and aren't necessarily using M3TSZ encoding dependening on the encoder pool specified.